halpybot/packages/board/board.py

```diff
@@ -53,9 +46,9 @@ def __init__(self, id_range):
     async def debug_load_board(self):
         """DEBUG: Load test data into the board"""
         self._cases_by_id = {
-            1: TempRescue(1, "bob"),
-            2: TempRescue(2, "larry"),
-            4: TempRescue(4, "john"),
+            1: Case(board_id=1, client_name="Bob", platform="PC", system="Delkar"),
+            2: Case(board_id=2, client_name="Larry", platform="PC", system="Delkar"),
+            4: Case(board_id=3, client_name="John", platform="PC", system="Delkar"),
```

Comment:
needs pytest fixture instead of this entire method

Reply:
This is all temporary stuff that will be removed later (or as soon as I can replicate the functionality externally) for live testing. I want to keep it in until then.